liblangutil/Scanner.cpp

```diff
@@ -349,18 +413,30 @@ size_t Scanner::scanSingleLineDocComment()
 
 Token Scanner::skipMultiLineComment()
 {
+	UnicodeDirectionOverrideProcessor unicodeDirectionalOverride{*this};
+
 	while (!isSourcePastEndOfInput())
 	{
-		char ch = m_char;
-		advance();
-
-		// If we have reached the end of the multi-line comment, we
-		// consume the '/' and insert a whitespace. This way all
-		// multi-line comments are treated as whitespace.
-		if (ch == '*' && m_char == '/')
+		auto const [processed, errorCode] = unicodeDirectionalOverride();
```

Comment:
Why can't `processed` be part of the enum?

Reply:
I am reusing `ScannerError` here, that can be "no-error" or some kind of error. This function however can have data processed regardless of whether or not an error was generated. I think I could introduce a new enum that this function will return and then map back, yes.